src/org/rascalmpl/library/List.rsc

```diff
@@ -96,8 +96,17 @@ dup([3, 1, 5, 3, 1, 7, 1, 2]);
 list[&T] dup(list[&T] lst) 
   = ([] | (ix in it) ? it : it + [ix] | &T ix <- lst);
 
-@deprecated{Use a list index instead}
```

Comment:
why is it undeprecated? don't we want to reduce the API size?

Reply:
I received feedback that people want to lei this function. I was trying to make this PR have fewer warnings in the stdlib so that's why this snuck in there. 

It's still strange to me that the declaration of a deprecated function leads to a warning on the declaration side btw. There is nothing wrong with a deprecated declaration. It's the uses which need a warning.